### PR TITLE
Change to onEvent

### PR DESCRIPTION
**Pull request by tiina303**
We're moving away from `exportEvents` to either batch exports or one event at a time webhooks (onEvent).

http://metabase-prod-us/question/272-braze-events-sent <- stats for how many events are processed together based on log lines in the code, but given that we then potentially break them up again it's hard to know how many events we typically actually send together. 

The app is simpler and doens't use the export bugger, which we suspect could be dropping events during restarts.

slack: https://posthog.slack.com/archives/C0386ASS3TN/p1694544727567379?thread_ts=1692342517.071949&cid=C0386ASS3TN

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -197,74 +197,27 @@ const _generateBrazeRequestBody = (pluginEvent: PluginEvent, meta: BrazeMeta): B
...
-    )
-
-    if (
-        brazeRequestBodies.length === 0 ||
-        brazeRequestBodies.every((body) => body.attributes.length === 0 && body.events.length === 0)
-    ) {
-        return console.log('No events to export.')
+    const brazeRequestBody = _generateBrazeRequestBody(pluginEvent, meta)
+    if (brazeRequestBody.attributes.length === 0 && brazeRequestBody.events.length === 0) {
+        return
```
Comment (igorosip0v) [inline index.ts:207]: @tiina303 @paolodamico Maybe it's a good idea to log here something:`Exit on the empty event: ${eventName}`

Comment (tiina303) [reply index.ts:207]: that could be quite noisy, no?

Comment (igorosip0v) [reply index.ts:207]: > that could be quite noisy, no?
Not sure, to be honest. How often it can be that a request contains no events and no attributes?

Comment (tiina303) [reply index.ts:207]: see https://posthog.slack.com/archives/C0386ASS3TN/p1695123530196279?thread_ts=1692342517.071949&cid=C0386ASS3TN for more discussion for your team's specifics

**Pull request closed without merging**